source/initial_composition/interface.cc

```diff
@@ -131,6 +131,18 @@ namespace aspect
     }
 
 
+
+    template <int dim>
+    void
+    Manager<dim>::update()
+    {
+      // Go through the list, and update plugins:
```

Comment:
```suggestion
      // Go through the list and update plugins:
```
or just get rid of these comments.